bftengine/src/bftengine/ReadOnlyReplica.cpp

```diff
@@ -211,6 +212,7 @@ void ReadOnlyReplica::executeReadOnlyRequest(concordUtils::SpanWrapper &parent_s
   const uint16_t clientId = request.clientProxyId();
 
   int status = 0;
+  int executionResult = 0;
```

Comment:
You don't need both - status and executionResult. They contain the same data. Remove status.